README.md

```diff
@@ -0,0 +1,21 @@
+# ITK Pretix
+
```

Comment:
A little more information on what this cool module actually does might be useful here.

Reply:
Added A little more information on what this cool module actually does